governator-core/src/main/java/com/netflix/governator/event/ApplicationEvent.java

```diff
@@ -0,0 +1,10 @@
+package com.netflix.governator.event;
+
+/**
+ * Marker interface for application events to be published
+ * to subscribed consumers.
+ */
+public interface ApplicationEvent {
```

Comment:
Exactly what I was thinking. Discovering what events exist and can be subscribed to is a pretty helpful thing for users. It could be shifted into an Annotation on the event object and found by references, but I don't think that changes much.